tracing-serde/Cargo.toml

```diff
@@ -17,9 +17,13 @@ categories = [
 ]
 keywords = ["logging", "tracing", "serialization"]
 
+[features]
+default = ["std"]
+std = ["serde/std", "tracing-core/std"]
```

Comment:
Can we add a list of feature flags in the crate's `lib.rs` docs and mention the existence of this one?